menu/router.js

```diff
@@ -0,0 +1,6 @@
+const { Router } = require("express");
+const router = new Router();
+
+const { Menu, Dish } = require("./model");
+
+module.exports = router;
```

Comment:
this router is not needed right?

Reply:
It will be... I'm currently working on the frontend part for this router.